### PR TITLE
Refactor random number functions

### PR DESCRIPTION
**Pull request by coryan**
Move some of the functions from `bigtable/benchmarks` to `bigtable/testing`.  I am planning to generate random names for instances and clusters, so we need less coordination in the continuous builds that run the integration tests.

### CODE REVIEW

**Comment by mbrukman** [comment]
@coryan — note that by adding `[skip ci]` to one of the commits, the entire PR appears to have skipped Travis CI testing — if you look at "All checks passed", it only shows the CLA passed.

Please consider either squashing the PR into a single commit, or at least removing the `[skip ci]` from the 3rd commit to force Travis CI to test this PR.

**Comment by coryan** [comment]
@mbrukman Well, that was silly.  Fixed with an empty commit.

**Comment by coryan** [comment]
@hemant-qlogic @manish-qlogic @roopak-qlogic: this is blocking the remaining fixes for #418, which would help with the integration tests and examples.  Please review when you get a chance.

**Comment by mbrukman** [comment]
@coryan — I was thinking more like `git commit --amend`, edit the last commit's description to remove `[skip ci]`, and `git push -f`, but whatever works for you. :-)

You can now squash the last 2 commits and remove the `[skip ci]` so that the history is cleaner, if you'd like.

File: bigtable/benchmarks/random_mutation_test.cc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2017 Google Inc.
+// Copyright 2018 Google LLC
```
Comment (mbrukman) [inline bigtable/benchmarks/random_mutation_test.cc]: Years should not be changed in copyright, but `Inc.` -> `LLC` is probably fine.

Comment (coryan) [reply bigtable/benchmarks/random_mutation_test.cc]: I "fixed" this, but the story is fun: the first commit in this PR created this file ( `bigtable/benchmarks/random_mutation_test.cc`), then I moved some code around and git decides that this is not a new file, it is me renaming a file and changing a few things, including the copyright line.  That is "clever" :frowning_face:

**Pull request merged**